SYCL/README.md

```diff
@@ -114,6 +114,9 @@ Defaults to AMD if no value is given. Supported values are:
  ***AMD_ARCH*** Flag must be set for when using HIP AMD triple.
  For example it may be set to "gfx906".
 
+***GPU_AOT_TARGET_OPTS*** - defines additional options which are passed to AOT
```

Comment:
Adding dash to separate variable name from it's description is a good idea. Please, apply the same style to the all variables above for consistency.